src/main/java/core/basesyntax/transactions/FruitTransaction.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.transactions;
```

Comment:
use single form for package names 
```suggestion
package core.basesyntax.transaction;
```